instance/tasks.py

```diff
@@ -73,6 +73,14 @@ def watch_pr():
                             .format(pr=pr, i=instance, truncated_title=truncated_title)
             instance.github_pr_url = pr.github_pr_url
             instance.ansible_extra_settings = pr.extra_settings
+            instance.ansible_source_repo_url = pr.extra_settings_dict.get('edx_ansible_source_repo')
+            if not instance.ansible_source_repo_url:
+                instance.ansible_source_repo_url = 'https://github.com/{}.git'.format(
```

Comment:
Nit/personal preference: I would rather have the default setting hold the entire repo url, rather than interpolating the fork/repo name.